pandas/core/ops.py

```diff
@@ -1762,6 +1764,7 @@ def wrapper(self, other, axis=None):
             return self._constructor(res_values, index=self.index,
                                      name=res_name, dtype='bool')
 
+    wrapper.__name__ = op_name
```

Comment:
is somthing testing these?

Reply:
No.  I have a local branch that checks these for all PandasObject subclasses, but there are a bunch of non-obvious cases in there.  This just fixes the more obvious cases.